service-front/module/Application/src/Model/Service/Lpa/Application.php

```diff
@@ -22,13 +22,11 @@
 use Opg\Lpa\DataModel\Lpa\Payment\Payment;
 use Opg\Lpa\DataModel\WhoAreYou\WhoAreYou;
 use ArrayObject;
-use Opg\Lpa\Logger\LoggerTrait;
```

Comment:
just checking if removal, non replacement was intentional

Reply:
logger is now inherited from AbstractService.